src/pretix/base/models/page.py

```diff
@@ -0,0 +1,48 @@
+from django.core.validators import RegexValidator
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from i18nfield.fields import I18nCharField, I18nTextField
+
+from pretix.base.models import LoggedModel
+
+
+class Page(LoggedModel):
+    title = I18nCharField(
+        verbose_name=_("Page title")
+    )
+    slug = models.CharField(
```

Comment:
Use [`SlugField`](https://docs.djangoproject.com/en/5.1/ref/models/fields/#slugfield), it already has the validator.